### ISSUE TITLE
Bug - JumpLinks - ARIA attributes missing in examples

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#6041]
**Describe the problem**
The examples for JumpLinks are missing the `aria-current="location"` attribute, which is required when a jump link item is selected. 

Additionally, we should consider updating examples that have a visible text label to use `aria-labelledby` instead of `aria-label`

**Expected behavior**

- Elements that have classes `pf-v5-c-jump-links__item pf-m-current` should have `aria-current="location"` added in
- Examples that are using a visible text label should remove `aria-label` and replace it with `aria-labelledby`, and give the label element an `id` to passed to the aria-labelledby attribute:
  ```html
  // Instead of this:
  <nav class="pf-v5-c-jump-links" aria-label="Jump to section">
  <div class="pf-v5-c-jump-links__main">
    <div class="pf-v5-c-jump-links__header">
      <div class="pf-v5-c-jump-links__label">Jump to section</div>

  // Use this:
  <nav class="pf-v5-c-jump-links" aria-labelledby="some-id">
  <div class="pf-v5-c-jump-links__main">
    <div class="pf-v5-c-jump-links__header">
      <div id="some-id" class="pf-v5-c-jump-links__label">Jump to section</div>
  ```
